src/lib.rs

```diff
@@ -5024,12 +5060,30 @@ pub struct Stats {
     /// The number of QUIC packets that were lost.
     pub lost: usize,
 
+    /// The number of QUIC packets with retransmitted data.
+    pub retrans: usize,
+
     /// The estimated round-trip time of the connection.
     pub rtt: time::Duration,
 
     /// The size of the connection's congestion window in bytes.
     pub cwnd: usize,
 
+    /// The number of sent bytes.
```

Comment:
It looks like in other cases (e.g. `recv_bytes` and `lost_bytes`) we count the total number of bytes in a packet (including the header), while `sent_bytes` only counts the payload. Is this on purpose? If so it needs to be documented I think.